src/brogue/Movement.c

```diff
@@ -2166,6 +2166,25 @@ boolean search(short searchStrength) {
     return foundSomething;
 }
 
+void manualSearch() {
+    short i, j, x, y;
+
+    // Guaranteed discovery of all adjacent locations
+    for (i = -1; i < 2; i++) {
+        x = player.xLoc + i;
+        for (j = -1; j < 2; j++) {
+            y = player.yLoc + j;
+            if (coordinatesAreInMap(x, y)) {
+              discover(x, y);
+            }
+        }
+    }
+
+    search(rogue.awarenessBonus + 30);
```

Comment:
Are we sure this is the search strength in 1.7.4? Looking at the original source (1b86334be874c78e4c663e890495d066d283e35b), I think this is used

```suggestion
    search(rogue.awarenessBonus < 0 ? 40 : 80);
```

Reply:
Oh right, good catch